### PR TITLE
Missing demos added,groups sorted alphabetically

### PR DESCRIPTION
**Pull request by Cy-Bo-Rg**
css3d_panorama_deviceorientation.html
webgl_geometry_spline_editor.html
webgl_materials_texture_hdr
webgl_objects_update
webgl_skinning_simple
canvas_lines_colors_2d

No more demos are missing.

### CODE REVIEW

**Comment by Cy-Bo-Rg** [comment]
I've moved a demo from its position because it wasn't sorted alphabetically but then I've seen that so many aren't sorted ,then I left the things as they were...  :P

**Comment by WestLangley** [comment]
>  I've seen that so many aren't sorted

I think alphabetical sorting within groups would be a good thing.

**Comment by Cy-Bo-Rg** [comment]
I can fix them if you don't want to mess with it :P
I recognize you're really busy with more important things ^_^"

**Comment by Cy-Bo-Rg** [comment]
"This pull request contains merge conflicts that must be resolved."
This is odd... :S

File: examples/index.html [hunk 1]
```diff
@@ -329,26 +332,27 @@
...
 				"webgl_shader",
-				"webgl_shader_lava",
 				"webgl_shader2",
 				"webgl_shaders_ocean",
 				"webgl_shaders_ocean2",
-				"webgl_shading_physical",
 				"webgl_shaders_sky",
 				"webgl_shaders_tonemapping",
 				"webgl_shaders_vector",
+				"webgl_shader_lava",
```
Comment (mrdoob) [inline examples/index.html]: I think this one was fine...?

**Comment by Cy-Bo-Rg** [comment]
I'm in a merge conflict situation but "git status" shows that everything is okay :s

File: examples/index.html (hunk 1 above)
Comment (Cy-Bo-Rg) [inline examples/index.html]: I've used a script..that's odd.. :P

Comment (Cy-Bo-Rg) [inline examples/index.html]: fixed.
I apologize.

**Comment by mrdoob** [comment]
Resolved the conflicts on my end. Thanks!

**Pull request merged**